### ISSUE TITLE
The data of humanml is of 153 dimension rather than 322 dimension.

### ISSUE THREAD

**Issue by weihaosky** [IDEA-Research/Motion-X#70]
Hi, thanks for this excellent work!

When I try to transfer the data to tomato representation with `python raw_pose_processing.py`, I find the data in face_motion_data/smplx_322/humanml or GRAB or EgoBody is of 153 dimension rather than 322 dimension, such that it cannot be processed by `python raw_pose_processing.py`.

How should I get the data of 322 dimension for dataset humanml, GRAB, and EgoBody?

Many thanks!

**Comment by linjing7** [comment]
Hi, have you preprocess the mocap data following this [instruction](https://github.com/IDEA-Research/Motion-X/blob/main/mocap-dataset-process/README.md)?

**Comment by weihaosky** [comment]
> Hi, have you preprocess the mocap data following this [instruction](https://github.com/IDEA-Research/Motion-X/blob/main/mocap-dataset-process/README.md)?

No, I download the motionx_face_motion_data.zip, which contains face_motion_data/smplx_322/humanml&GRAB&EgoBody
I did not do the preprocess from AMASS dataset.
